### PR TITLE
Migration guide

### PR DESCRIPTION
**Pull request by darko-mijic**

### CODE REVIEW

**Comment by DominikGuzei** [comment]
@darko-mijic the guide is updated for 0.6.0 -> 0.7.0 and i removed the old cruft

**Comment by darko-mijic** [comment]
@DominikGuzei: Change mgp config for testing-flux to `9b91be5efbb2adcb86b461f44a16d8f7ca44111d`

**Comment by darko-mijic** [comment]
also in package.js `'space:testing-flux@1.0.0'` -> `'space:testing-flux@0.7.0'`

**Comment by darko-mijic** [comment]
Maybe `Space.flux.Reactive` needs to be mentioned in the readme...

**Comment by DominikGuzei** [comment]
That's your job then :wink: im leaving now! Will be back tomorrow.

**Comment by darko-mijic** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -82,189 +85,61 @@ on one or more minimongo collections in your app and constructs queries based
...
-2. If you need to manage state that you don't want to hold in a collection
-you can use the new API to generate `ReactiveVar` instance accessors:
-```javascript
-Space.flux.Store.extend(TodoMVC, 'TodosStore', {
-  reactiveVars: function() {
-    return [{
-      activeFilter: this.FILTERS.ALL,
-    }];
+// 0.7.0 example
+Space.ui.Mediator.extend('TodoMVC.TodoListMediator', {
```
Comment (darko-mijic) [inline README.md]: No mediators :)

**Pull request closed without merging**